app/views/helpers/lists.php

```diff
@@ -144,15 +160,24 @@ public function displayRow(
             );
             echo format(__('created by {listAuthor}', true),
                         array('listAuthor' => $link));
+            
+            echo $this->Html->tag(
+                'span',
+                ' '.$this->Date->ago($createdDate),
+                array(
+                    'class' => 'date'
+                )
+            );
```

Comment:
Hmm, I'm not following anymore... Maybe I was not very clear in my previous comments. 

In terms of code, what I meant was that instead of:
```
$link = $this->Html->link(
    $listCreatorName,
    array(
        "controller"=>"user",
        "action"=>"profile",
        $listCreatorName
    )
);
echo format(__('created by {listAuthor}', true),
            array('listAuthor' => $link));

echo $this->Html->tag(
    'span',
    ' '.$this->Date->ago($createdDate),
    array(
        'class' => 'date'
    )
);
```

to have:
```
$link = $this->Html->link(
    $listCreatorName,
    array(
        "controller"=>"user",
        "action"=>"profile",
        $listCreatorName
    )
);
$creationDate = $this->Html->tag(
    'span',
    ' '.$this->Date->ago($createdDate),
    array(
        'class' => 'date'
    )
);
echo format(
    __('created by {listAuthor}, {date}', true),
    array(
        'listAuthor' => $link,
        'date' => $creationDate
    )
);
```

Your adding a comma after `{listAuthor}` works too, but it's just a bit less "translator-friendly" because it could happen that in this context, the translator needs to add some words before or after `{date}`, and they wouldn't be able to do it if the string was only `created by {listAuthor},`.